recipe/meta.yaml

```
@@ -7,21 +7,21 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  number: 2
```

```suggestion
  number: 1
```
I don't see build number 1 published